ar4_mujoco_sim/README.md

```diff
@@ -0,0 +1,12 @@
+# Description
```

Comment:
This is missing the instructions to launch the simulator.

Reply:
Fixed.